slidez-core/src/main/java/com/binarystudio/academy/slidez/domain/session/SessionService.java

```diff
@@ -0,0 +1,80 @@
+package com.binarystudio.academy.slidez.domain.session;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import javax.persistence.EntityNotFoundException;
+
+import com.binarystudio.academy.slidez.domain.presentation.PresentationService;
+import com.binarystudio.academy.slidez.domain.presentation.model.Presentation;
+import com.binarystudio.academy.slidez.domain.session.dto.SessionUpdateDto;
+import com.binarystudio.academy.slidez.domain.session.model.Session;
+import com.binarystudio.academy.slidez.domain.session.model.SessionStatus;
+import static java.time.LocalDateTime.now;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SessionService {
+
+	@Autowired
+	private SessionRepository sessionRepository;
+
+	@Autowired
+	private PresentationService presentationService;
+
+	public Session add(Presentation presentation, SessionStatus status) {
```

Comment:
How will this method be used? 
It's generally a bad practice to add non DTO or non primitive parameters to public service methods

Reply:
Redundant method removed.